src/Native/configure.cmake

```diff
@@ -317,6 +317,14 @@ check_symbol_exists(
     "sys/syslimits.h"
     HAVE_OPEN_MAX)
 
+if (CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
Probably @lhoward (from https://github.com/heimdal/heimdal/) can answer that with surety, who implemented Windows CNG in heimdal.

According to https://wiki.ncsa.illinois.edu/display/ITS/Kerberos+on+Mac+OS+X+10.7+and+later:

> Heimdal Kerberos is shipped as part of Mac OS X (as of the OS X 10.7 "Lion" release).

So the underlying lib might be available with a different name

Also lib seems to be built as part of freebsd source: https://github.com/freebsd/freebsd/tree/master/kerberos5/lib/libheimntlm.